### PR TITLE
Browsing to unknown page causes fatal error

### PR DESCRIPTION
**Pull request by scottcwilson**
Example: browse to index.php?main_page=unknown 

[10-Aug-2020 11:28:24 UTC] Request URI: /gh_demo_158/index.php?main_page=unknown, IP address: ::1
#1  require() called at [/Users/scott/Sites/gh_demo_158/includes/templates/responsive_classic/common/tpl_main_page.php:177]
#2  require(/Users/scott/Sites/gh_demo_158/includes/templates/responsive_classic/common/tpl_main_page.php) called at [/Users/scott/Sites/gh_demo_158/index.php:94]
--> PHP Warning: require(includes/templates/template_default/templates/tpl_unknown_default.php): failed to open stream: No such file or directory in /Users/scott/Sites/gh_demo_158/includes/templates/responsive_classic/common/tpl_main_page.php on line 177.

[10-Aug-2020 11:28:24 UTC] PHP Fatal error:  require(): Failed opening required 'includes/templates/template_default/templates/tpl_unknown_default.php' (include_path='.:/Applications/MAMP/bin/php/php7.3.9/lib/php') in /Users/scott/Sites/gh_demo_158/includes/templates/responsive_classic/common/tpl_main_page.php on line 177
[10-Aug-2020 11:28:24 UTC] PHP Stack trace:
[10-Aug-2020 11:28:24 UTC] PHP   1. {main}() /Users/scott/Sites/gh_demo_158/index.php:0
[10-Aug-2020 11:28:24 UTC] PHP   2. require() /Users/scott/Sites/gh_demo_158/index.php:94

### CODE REVIEW

**Comment by lat9** [comment]
Shouldn't this condition have been caught by `/includes/init_includes/init_sanitize.php`, like in previous Zen Cart versions?  There's also that configuration setting that indicates whether page-not-found should go to the main page or (er) the page-not-found one.

**Comment by scottcwilson** [comment]
Good point - updated.

File: includes/languages/english.php [hunk 1]
```diff
@@ -570,7 +570,7 @@ function zen_date_raw($date, $reverse = false) {
 
   define('DB_ERROR_NOT_CONNECTED', 'Error - Could not connect to Database');
   define('ERROR_DATABASE_MAINTENANCE_NEEDED', '<a href="https://docs.zen-cart.com/user/troubleshooting/error_71_maintenance_required/" rel="noopener" target="_blank">ERROR 0071 There appears to be a problem with the database. Maintenance is required.</a>');
-
+  define('ERROR_PAGE_NOT_FOUND', 'Sorry, the page you were attempting to access cannot be found.');
```
Comment (drbyte) [inline includes/languages/english.php:573]: Is this additional language define necessary? 

And doesn't it duplicate the one in page.php which will therefore cause an Already Defined notice?

Comment (scottcwilson) [reply includes/languages/english.php:573]: Yes, you're right - it's a relic of the first approach.

File: includes/languages/english.php [hunk 2]
```diff
@@ -570,7 +570,7 @@ function zen_date_raw($date, $reverse = false) {
 
   define('DB_ERROR_NOT_CONNECTED', 'Error - Could not connect to Database');
   define('ERROR_DATABASE_MAINTENANCE_NEEDED', '<a href="https://docs.zen-cart.com/user/troubleshooting/error_71_maintenance_required/" rel="noopener" target="_blank">ERROR 0071 There appears to be a problem with the database. Maintenance is required.</a>');
-
+
```
Comment (drbyte) [inline includes/languages/english.php:573]: Sorry to be picky, but your last update still has the lang file updated by adding spaces.

Comment (scottcwilson) [reply includes/languages/english.php:573]: Can you fix this for me?  I'm away from my home office.

Comment (scottcwilson) [reply includes/languages/english.php]: this should be fixed now.

**Review by zcwilt — APPROVED** [review]

**Pull request merged**